docs/usage/explicit-reference.md

```diff
@@ -36,3 +36,25 @@ export class AppComponent {
   faCoffee = faCoffee;
 }
 ```
+
+You can also use a [https://fontawesome.com/kits](Font Awesome Kit). With a Kit
```

Comment:
Bad markdown, the URL and label should be swapped

<img width="1003" alt="image" src="https://github.com/FortAwesome/angular-fontawesome/assets/823594/edcb6fe1-5026-4649-9834-8d79e9832085">